tornjak-frontend/src/components/navbar.tsx

```diff
@@ -70,6 +122,43 @@ class NavigationBar extends Component<NavigationBarProp, NavigationBarState> {
                 }}
               >Tornjak Dashboard</a>
             </div>
+            <div className='header-toolbar'>
+              {Auth_Server_Uri &&
+                <div className="user-dropdown">
+                  <HeaderGlobalAction
+                    aria-label="User">
+                    <UserAvatar20 />
+                  </HeaderGlobalAction>
+                  <div className="user-dropdown-content">
+                    {KeycloakService.isLoggedIn() && (
+                      // eslint-disable-next-line
+                      <a
+                        href="#"
+                        className="nav-link"
+                        onClick={() => KeycloakService.doLogout()}>
+                        Logout {KeycloakService.getFirstName()}
+                      </a>
+                    )}
+                  </div>
+                </div>
+              }
+              <HeaderGlobalAction
+                aria-label="Notifications"
+                onClick={() => { alert("This is a place holder, functionality to be implemented on future work!") }}>
+                <Notification20 />
+              </HeaderGlobalAction>
+              <HeaderGlobalAction
+                aria-label="Search"
+                onClick={() => { alert("This is a place holder, functionality to be implemented on future work!") }}>
+                <Search20 />
+              </HeaderGlobalAction>
```

Comment:
What are these placeholders for?

Reply:
These are search functionality and notification functionality for Tornjak UI. Comes with the Header Global Action component, when we want to implement those functionalities in the future it's there.